user/pages/04.Reference/05.object-storage/docs.en.md

```diff
@@ -137,6 +138,54 @@ where REGION is the SysEleven Stack region (e.g. cbk or dbl).
 
 You can use these URLs to refer to the uploaded files as static assets in your web applications.
 
+If you plan to use more complex ACLs we suggest you checkout the boto3 client.
```

Comment:
This line looks like a bridge passage but imo rather belongs to the boto3 section, than to the s3cmd section. I suggest to move it behind the section headline.

Reply:
Alright! After we have fully reworked the ACL tutorial we may also rework the background article as we might have duplicate information between both articles

Reply:
The line was already removed and the boto3 sections contains the hint it would be tool of choice for more complex ACL scenarios